ui/_tests_/utils/camel_to_sentence.test.js

```diff
@@ -0,0 +1,8 @@
+import { CamelCaseToSentanceCase } from "../../utils/camelCaseToSentanceCase";
```

Comment:
we have a convention of naming the file like `testNameinCamelCase_spec.js`

Reply:
Gotcha.